powersimdata/input/input_data.py

```diff
@@ -25,6 +28,8 @@ def __init__(self, data_loc=None):
             "ct": "pkl",
             "grid": "mat",
         }
+        self.profile = {"demand", "hydro", "solar", "wind"}
```

Comment:
This feels like a module level constant

Reply:
Sounds good. I will declare it outside of the class

Reply:
Done